### DISCUSSION TITLE
littlelink Remedies LinkTree's SEO Issues?

### DISCUSSION THREAD

**Discussion by dm17** [sethcottle/littlelink#75] [General]
Hopefully this is an acceptable place to have a discussion...
[There are well known SEO problems with using LinkTree](https://studiocotton.co.uk/blog/linktree-bad-seo-alternative/). littlelink can fix the SEO issue of not owning the domain you're using for links, but does anyone know how much that remedies the overall SEO issue of this "one link" approach? Perhaps there are littlelink users who have done SEO investigations that can share optimizations they've used or any downsides or fixes they've experience from self-hosting on their own domain name.
Thanks!

**Comment by sethcottle** [comment]
The biggest benefits come from having control over the meta tags, domain, and analytics in my opinion. You can perform so many optimizations through your meta tags to help with your rankings and you'll be able to gather much more in-depth metrics that you can get from LinkTree (especially their free plan) depending on the analytics route you want to take. 

There are obvious draw backs as well to pages like this, even with LinkTree, having a single page that users instantly direct themselves off of will increase your bounce rate, which to my knowledge makes some crawlers not too pleased since visitors are spending less time on your site.

But I inherently believe pages like this just aren't the SEO generators people desire them to be. You're making an all-in-one page for the convenience of a user and improving the experience for users who are likely coming from a social profile where you can only have one link to get them to all of your stuff. While you can still optimize and snag yourself some SEO benefits from this, ideally your main website should be doing the SEO work for you.
